src/App.tsx

```diff
@@ -1,16 +1,23 @@
+import { RouterProvider } from "react-router";
+import { router } from "./routes/router";
+import { DragDropContext, DropResult } from "@hello-pangea/dnd";
+import { useCallback } from "react";
+import { useTaskDrag } from "./components/tasks/hooks/useTaskDrag";
+
 function App() {
+  const { handleDragEnd } = useTaskDrag();
+  const onDragEnd = useCallback((result: DropResult) => {
+    handleDragEnd(result);
+  }, []);
```

Comment:
handleDragEnd is a reactive value, needs to be added to the dependency array:

```suggestion
  }, [handleDragEnd]);
```

source: https://react.dev/reference/react/useEffect#specifying-reactive-dependencies, it talks about the useEffect hook, but the same applies for every hook that has a dependency array.

Reply:
Can we add a Eslint rule to warns us about missing dependency array values please?